src/tools/base/BaseAnnotationTool.js

```diff
@@ -5,6 +5,9 @@ import {
   moveHandleNearImagePoint,
   moveAnnotation,
 } from './../../util/findAndMoveHelpers.js';
+import { getLogger } from '../../util/logger';
+
+const logger = getLogger('addTool');
```

Comment:
https://github.com/cornerstonejs/cornerstoneTools/blob/master/src/util/logger.js#L29-L39

I'm pretty new to our logging updates, but we may want a scope that's more specific to the module we're in. Maybe `baseAnnotationTool` or something equally as namespace-y?